hs_core/views/__init__.py

```diff
@@ -380,7 +380,7 @@ class FilterForm(forms.Form):
     start = forms.IntegerField(required=False)
     published = forms.BooleanField(required=False)
     edit_permission = forms.BooleanField(required=False)
-    creator = forms.ModelChoiceField(queryset=User.objects.all(), required=False)
```

Comment:
The hs_core.views.FilterForm itself is not used, but the REST serializers do something similar, and support the creator field.  I just wanted to make sure we were not introducing/removing a capability to/from the web client that is not/no longer present in the REST API.